differ/differ.go

```diff
@@ -368,7 +477,12 @@ func processReportsByCluster(ruleContent types.RulesMap, storage Storage, cluste
 		}
 
 		log.Info().Msgf("Producing instant notification for cluster %s with %d events", string(cluster.ClusterName), len(notificationMsg.Events))
-		_, _, err = notifier.ProduceMessage(notificationMsg)
+		msgBytes, err := json.Marshal(notificationMsg)
+		if err != nil {
+			log.Error().Err(err).Msg(invalidJSONContent)
+			os.Exit(ExitStatusMessageEncoderError)
```

Comment:
so it exit even because one wrong message?

Reply:
Replaced exit with continue